helmfile/values/dex.yaml.gotmpl

```diff
@@ -28,6 +28,8 @@ volumes:
 {{- end }}
 
 config:
+  expiry:
+    idTokens: {{ .Values.dex.idTokenExpiryTime }}
```

Comment:
Should we add all the expiry times when were at it?

Reply:
Do you see any cases where the other ones could be relevant? I think some of these would just cause confusion, like `signingKeys` and `deviceRequests` https://github.com/dexidp/dex/blob/master/examples/config-dev.yaml#L78-L85

Reply:
I think it is enough with just being able to configure the id token

Reply:
At least the refresh tokens and signing keys are involved in the flow. But I'd rather have them configurable, so when we get to the situation where we need to modify them we already have it instead of needing to wait for another release.

Reply:
Make sense. Then please @davidumea add configuration options for those as well